lib/active_record/connection_adapters/postgis/version.rb

```diff
@@ -3,7 +3,7 @@
 module ActiveRecord
   module ConnectionAdapters
     module PostGIS
-      VERSION = "8.0.2"
+      VERSION = "9.0.0"
```

Comment:
We'll bump version when pushing to the repository, no need to do it in a PR!

Reply:
Sure. I was testing it in live apps. I will revert